l10n_it_fatturapa_out/wizard/__init__.py

```diff
@@ -7,6 +7,6 @@
 # there are multiple statements that should be guarded
 # in wizard_export_fatturapa, so we guard the entire file
 try:
-    from . import wizard_export_fatturapa
+    from . import wizard_export_fatturapa_regenerate  # noqa
```

Comment:
Come mai `noqa`?

Reply:
Evita di mostrare l'errore imported but not used

Reply:
Come per `l10n_it_fatturapa_pec/__init__.py`